.github/workflows/qe-hosted.yml

```diff
@@ -13,6 +13,10 @@ permissions:
 
 env:
   QE_REPO: test-network-function/cnfcert-tests-verification
+  TEST_TNF_IMAGE_NAME: quay.io/testnetworkfunction/cnf-certification-test
```

Comment:
What about using `CERTSUITE_IMAGE_NAME` instead of `TEST_TNF_IMAGE_NAME ` .

The same applies to the names below.

Reply:
Good call I'll update.